### PR TITLE
[GLOBAL-146] - Add red colours

### PR DESCRIPTION
**Pull request by tpgmartin**
@nickbalestra

### CODE REVIEW

File: OTTheme/aliases.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@ aliases:
     value: "#E5656E"
   color-black-cherry:
     value: "#811520"
+  color-warning-red:
+    value: "#EFD4D4"
+  color-content-highlighted-red:
+    value: "#FEF8F0"
```
Comment (karuto) [inline OTTheme/aliases.yml]: People seem to dislike functional color naming in https://github.com/opentable/design-tokens/issues/52

**Comment by tpgmartin** [comment]
@karuto let me know if this works, thanks

**Comment by tpgmartin** [comment]
@karuto I've been given the seal of approval from Sim

**Comment by karuto** [comment]
@tpgmartin I merged, can you publish a patch version for this token?

**Pull request merged**